Update SAP Hana Cloud Platform

stefan-kolb opened the pull request:
Based on your email and http://de.slideshare.net/saphcp/sap-hana-cloud-platform-product-road-map-q32015:

Removed:
- Data center Amsterdam as it is marked for disaster recovery only
- No planned innovations yet, please (e.g. Cloud Foundry runtimes). Only features that are in production.

gdsap commented on profiles/sap_hana_cloud.json:
```diff
@@ -1,6 +1,7 @@
 {
   "name": "SAP HANA Cloud",
```
The name should be SAP HANA Cloud Platform.

gdsap commented:
Hi Stefan,
1. Line no: 2 The name should be SAP HANA Cloud Platform.
2. Line no: 7 There should be one more type Paas centric. Keeping in mind that VM and CF are not allowed.
3. Line no: 44 & 47, Middleware and framework are missing, I have mentioned them in the document

Thanks.

stefan-kolb commented:
Hi,

1. I changed the name of the profile.
2. There is no PaaS-centric. Please see https://github.com/stefan-kolb/paas-profiles#type for details. At the moment I'd say it is more SaaS-centric than Generic. With the addition of CF and VMs this might change.
3. Added Java EE as framework. Added some middleware products. This category should list middleware technologies dependent on runtimes not middleware concepts realizable in every runtime, like SOAP or REST. CloudConnector is already listed as service

I know some things are debatable, especially since SAP Hana Cloud platform is not an ordinary PaaS.

gdsap commented:
For framework add java web and java web profile.I think by mistake you added it under middleware. In Middleware: HANA Cloud Integration will do.

stefan-kolb commented:
What's the difference between Java EE web and java web profile? I only know Java EE Web profile. Will add Hana Cloud integration.

gdsap commented:
Typo. Java EE and Java web profile

stefan-kolb commented:
Ok, I think we are good to go.

gdsap commented:
Great !

gdsap commented:
One small change : Remove Java EE from middleware add it to the framework along with: Java Web Profile.

It should look like:

Middleware
Name	Runtime	Versions
Hana cloud integration		
==================================
Frameworks
Name	Runtime	Versions
Java web profile   ll	Java	
Java EE        ll     Java    


Thanks.

gdsap commented:
One small change : Remove Java EE from middleware add it to the framework along with: Java Web Profile.

It should look like:

Middleware
Name Runtime Versions
HANA Cloud Integration

Frameworks
Name Runtime Versions
Java web profile  Java
Java EE  Java

Thanks.

stefan-kolb commented:
For me Java EE is not a framework but a middleware.

gdsap commented:
If you see this: 
https://glassfish.java.net/webprofileORfullplatform31x.html 
http://stackoverflow.com/questions/24239978/java-ee-6-web-profile-vs-java-ee-6-full-platform

Also the message that we convey we mention that these two are more of a frameworks rather than middleware. 
https://help.hana.ondemand.com/help/frameset.htm?7613bd28711e1014839a8273b0e91070.html

And the site shows "SAP HANA" in lowercase under the services section which should be uppercase everywhere. Even "API", "ASE", "GIS", "IoT",

stefan-kolb commented:
I know the differences between Java EE and the web profile, but most people would categorize Java EE as a middleware product. I'll change it for you anyway.

I cannot change your second request at the moment as the logic lowercases everything right now. This is a change not related to the pull request of the profile.

gdsap commented:
Hi Stefan,
Thanks for updating the middleware and framework. Regarding the second issue this will be a trademark problem. As it should be "SAP" and so on as mentioned above.

stefan-kolb commented:
I will have a look at this in the next days.
Am 01.04.2016 9:29 vorm. schrieb "gdsap" <notifications@github.com>:

> Hi Stefan,
> Thanks for updating the middleware and framework. Regarding the second
> issue this will be a trademark problem. As it should be "SAP" and so on as
> mentioned above.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/stefan-kolb/paas-profiles/pull/161#issuecomment-204290073>
>

gdsap commented:
Great. Just let me know when it's done.

gdsap commented:
Hey Stefan,

Can you have a look at the lowercase issues for "SAP HANA" ,"API", "ASE", "GIS", "IoT",
Kindly refer the pdf that I have sent across for better quality. These are trademark symbols.

gdsap commented:
Thanks, the issues are resolved.
